docs/architecture-description.md

```diff
@@ -577,52 +580,41 @@ Compression proof request file format:
 
 ```
 BlobCompressionProofJsonRequest
```

Comment:
what is in BlobJsonFileRequestResponse.kt is different from what is updated here

Reply:
yes, done!